### DISCUSSION TITLE
[Help Wanted] Display Line numbers

### DISCUSSION THREAD

**Discussion by drohm** [standardnotes/forum#2683] [Suggestions]
**Describe the issue**
Is it possible to show line numbers for some of the note types, such as Plain Text, Rich Text, Alternative Rich Text, etc?  The only note type that I've found to be able to show line numbers is the Code note type.

**Desktop (please complete the following information):**
- OS: Fedora 36 KDE
- Browser Brave v1.44.108
- AppImage 3.23.193

**Comment by effieeee** [comment]
Hi @drohm, I'm afraid that isn't possible at the moment, so I'll be marking this as an enhancement! As the Rich Text note types are from 3rd party sources, it may be more difficult to have that implemented, but we can at least look into this for Plain Text!
